Mathlib/SetTheory/Cardinal/Aleph.lean

```diff
@@ -483,6 +483,8 @@ theorem aleph1_eq_lift {c : Cardinal.{u}} : ℵ₁ = lift.{v} c ↔ ℵ₁ = c :
 theorem lift_eq_aleph1 {c : Cardinal.{u}} : lift.{v} c = ℵ₁ ↔ c = ℵ₁ := by
   simpa using lift_inj (b := ℵ₁)
 
+theorem aleph_eq_aleph' (o : Ordinal) : ℵ_ o = preAleph (ω + o) :=
```

Comment:
This theorem is a duplicate of [`Cardinal.aleph_eq_preAleph`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/SetTheory/Cardinal/Aleph.html#Cardinal.aleph_eq_preAleph) and it should have been deprecated.

Reply:
@vihdzp, would you mind just pushing this change to this PR?